src/app/admin/_components/isActive.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { BooleanInput } from 'react-admin';
+import PropTypes from 'prop-types';
+
+const IsActive = ({ onChange, label, className }) => (
+  <BooleanInput
+    name="isActive"
+    source="isActive"
+    label={label}
+    onChange={onChange}
+    className={className}
+  />
```

Comment:
I dont see much sense to have this component, it saves only 2 lines of code name="isActive" and source="isActive", level, onChange, className come from props anyway

Reply:
Removed this component